include/emp/prefab/ValueBox.hpp

```diff
@@ -0,0 +1,182 @@
+#ifndef EMP_VALUE_BOX_HPP
+#define EMP_VALUE_BOX_HPP
+
+#include <cmath>
+
+#include "../datastructs/set_utils.hpp"
+#include "../tools/string_utils.hpp"
+
+#include "../web/Div.hpp"
+#include "../web/Element.hpp"
+#include "../web/Input.hpp"
+
+#include "Collapse.hpp"
+#include "CommentBox.hpp"
+#include "FontAwesomeIcon.hpp"
+#include "ToggleSwitch.hpp"
+
+namespace emp::prefab {
+
+  const int min_slider_width = 100;
+
+  class ValueBox : public web::Div {
```

Comment:
Needs copyright etc boilerplate up at very top and a docstring explaining the class

Reply:
make a pass for this and docstrings throughout;

Reply:
Still needs fix?